tests/data/match/mozart_k265_var1.match

```diff
@@ -3,300 +3,227 @@ info(piece,mozart_k265_var1).
-snote(n191,[G,n],5,20:2,3/16,1/16,39.7500,40.0000,[v1,staff1])-note(n189,79,18948,18977,71,1,1).
-snote(n195,[C,#],4,20:2,3/16,1/16,39.7500,40.0000,[v5,staff2])-note(n188,61,18940,18986,58,1,1).
-snote(n196,[G,n],5,21:1,0,1/16,40.0000,40.2500,[v1,staff1])-note(n190,79,19065,19157,77,1,1).
-snote(n204,[D,n],4,21:1,0,1/4,40.0000,41.0000,[v5,staff2])-note(n191,62,19087,19522,74,1,1).
-snote(n197,[F,n],5,21:1,1/16,1/16,40.2500,40.5000,[v1,staff1])-note(n192,77,19185,19289,84,1,1).
-snote(n198,[D,n],6,21:1,1/8,1/16,40.5000,40.7500,[v1,staff1])-note(n193,86,19293,19411,75,1,1).
-snote(n199,[C,n],6,21:1,3/16,1/16,40.7500,41.0000,[v1,staff1])-note(n194,84,19409,19543,62,1,1).
-snote(n200,[B,n],5,21:2,0,1/16,41.0000,41.2500,[v1,staff1])-note(n195,83,19506,19618,71,1,1).
-snote(n201,[A,n],5,21:2,1/16,1/16,41.2500,41.5000,[v1,staff1])-note(n196,81,19616,19671,45,1,1).
-snote(n202,[G,n],5,21:2,1/8,1/16,41.5000,41.7500,[v1,staff1])-note(n197,79,19679,19734,70,1,1).
-snote(n203,[F,n],5,21:2,3/16,1/16,41.7500,42.0000,[v1,staff1])-note(n198,77,19810,19845,81,1,1).
-snote(n206,[B,n],3,21:2,3/16,1/16,41.7500,42.0000,[v5,staff2])-note(n199,59,19840,19876,47,1,1).
-snote(n207,[F,n],5,22:1,0,1/16,42.0000,42.2500,[v1,staff1])-note(n200,77,19948,20032,77,1,1).
-snote(n215,[C,n],4,22:1,0,1/4,42.0000,43.0000,[v5,staff2])-note(n201,60,19981,20353,69,1,1).
-snote(n208,[E,n],5,22:1,1/16,1/16,42.2500,42.5000,[v1,staff1])-note(n202,76,20053,20156,77,1,1).
-snote(n209,[C,n],6,22:1,1/8,1/16,42.5000,42.7500,[v1,staff1])-note(n203,84,20176,20270,68,1,1).
-snote(n210,[B,n],5,22:1,3/16,1/16,42.7500,43.0000,[v1,staff1])-note(n204,83,20276,20417,62,1,1).
-snote(n211,[A,n],5,22:2,0,1/16,43.0000,43.2500,[v1,staff1])-note(n205,81,20379,20485,59,1,1).
-snote(n212,[G,n],5,22:2,1/16,1/16,43.2500,43.5000,[v1,staff1])-note(n206,79,20462,20534,71,1,1).
-snote(n213,[F,n],5,22:2,1/8,1/16,43.5000,43.7500,[v1,staff1])-note(n207,77,20557,20635,72,1,1).
-snote(n214,[E,n],5,22:2,3/16,1/16,43.7500,44.0000,[v1,staff1])-note(n208,76,20663,20743,76,1,1).
-snote(n217,[A,n],3,22:2,3/16,1/16,43.7500,44.0000,[v5,staff2])-note(n209,57,20675,20787,61,1,1).
-snote(n218,[D,n],5,23:1,0,1/8,44.0000,44.5000,[v1,staff1])-note(n210,74,20796,21067,64,1,1).
-snote(n222,[F,n],3,23:1,0,1/4,44.0000,45.0000,[v5,staff2])-note(n211,53,20854,21113,72,1,1).
-snote(n219,[A,n],5,23:1,1/8,1/8,44.5000,45.0000,[v1,staff1])-note(n212,81,21064,21476,68,1,1).
-snote(n220,[G,n],5,23:2,0,1/8,45.0000,45.5000,[v1,staff1])-note(n214,79,21296,21543,75,1,1).
-snote(n223,[G,n],3,23:2,0,1/4,45.0000,46.0000,[v5,staff2])-note(n213,55,21287,21555,70,1,1).
-snote(n221,[B,n],4,23:2,1/8,1/8,45.5000,46.0000,[v1,staff1])-note(n215,71,21540,21748,58,1,1).
-snote(n224,[C,n],5,24:1,0,1/4,46.0000,47.0000,[v1,staff1])-note(n216,72,21793,22915,48,1,1).
-snote(n226,[C,n],4,24:1,0,1/4,46.0000,47.0000,[v5,staff2])-note(n217,60,21819,22029,49,1,1).
-snote(n227,[C,n],3,24:2,0,1/4,47.0000,48.0000,[v5,staff2])-note(n218,48,22393,22966,50,1,1).
-sustain(0,0).
```

Comment:
Why were the sustain pedal lines deleted from the file?

Reply:
Good catch! In another commit I changed the midi import to have each performed note have an int ID starting from 1 (as opposed to 0), probably the file got overwritten at some point back then. Changed it back, now the pedal lines are included again.